packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,183 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea, TextareaProps } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import clsx from 'clsx';
+import React from 'react';
+import { designTokenStory } from './util';
+
+interface TextareaStoryProps extends TextareaProps {
+  focus?: boolean;
+  focusVisible?: boolean;
+  hover?: boolean;
+  children?: any;
+}
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const TextareaStory = ({ children, focus, focusVisible, hover, ...args }: TextareaStoryProps) => {
+  const classNames = {
+    'utrecht-textarea--focus': focus,
+    'utrecht-textarea--focus-visible': focusVisible,
+    'utrecht-textarea--hover': hover,
+  };
+
+  return <Textarea className={clsx(classNames)} defaultValue={children} {...args} />;
+};
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: TextareaStory,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+    focus: {
+      description: 'Focus',
+      control: 'boolean',
+    },
+    focusVisible: {
+      description: 'focusVisible',
+      control: 'boolean',
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
```

Comment:
De `placeholder` property is verdwenen.

## Main branch

<img width="1037" alt="Screenshot 2023-09-05 at 08 10 44" src="https://github.com/nl-design-system/utrecht/assets/30694/6c843ba5-9eef-401e-8f35-a077a29a0a66">

## Deze branch

<img width="1031" alt="Screenshot 2023-09-05 at 08 07 38" src="https://github.com/nl-design-system/utrecht/assets/30694/9ac5e7e8-fa95-4ecf-bc6c-fe50f45bfb5e">